scripts/upgrade

```diff
@@ -29,6 +29,15 @@ if [[ -z "${secret_key:-}" ]]; then
 	ynh_app_setting_set --app=$app --key=secret_key --value=$secret_key
 fi
 
+#=================================================
+# BACKUP SQLITE DATABASE
+#=================================================
+ynh_script_progression --message="Backup of the DB in temporary file...." --weight=5
+
+# copy data directory backup
+tmp_data_backup=$(mktemp -d $app_dataXXX --suffix=BKP)
+rsync -R $install_dir/data/ /var/www/$tmp_data_backup/
```

Comment:
(I insist that the point of `mktemp -d` is that it already creates a directory, it doesn't make sense to use `/var/www/$tmp_data_backup` as the destination ... use `$tmp_data_backup` directly ... Like, try `mktemp -d` on your own machine ... You will see that it returns something like `/tmp/tmp.B6HJExzyJv` and `ls -ld /tmp/tmp.B6HJExzyJv` shows that the directory exists and has proper secure permissions)

Reply:
Ok I try understood, thanks

Reply:
I can't resolved tryng in console how use `rsync` to sync the folder `/var/www/$app/data/` with `$tmp_data_backup`
I am tryng on a test environment how to do...

Reply:
Ok I need use another command like `rsync -a`

Reply:
Ok, this time it works... Is it correct for you @alexAubin ?

Reply:
I guess that looks okay but honestly I'm lost, there is just too many PRs with overlapping commits ... I don't know why you created a "fix-testing" branch, testing is already a branch and is not merged, you could fix testing ... directly in `testing` itself x_x

The main goal now should be to merge or close PRs #33 and #34, and resolve conflicts between master and testing (in PR #24) such that we can get back on track with some proper clean branch ...

Reply:
I am really sorry for all these PRs, I want merge first the PR #33 on master branch, then I will resolve also the PR #34. 
For the PR #24, I wait a little, this version is very new and some bugs are resolved each weeks, it can let me time to resolved conflicts also for a proper branch...